src/file_organizer/cli/main.py

```
@@ -261,9 +450,30 @@ def history(
         operation_type=operation_type,
         status=status,
         stats=stats,
-        verbose=verbose or _g.verbose,
+        verbose=_merge_flag(verbose, _get_state().verbose),
     )
-    raise typer.Exit(code=code)
+    raise typer.Exit(code=code if code is not None else 1)
+
+
+@app.command()
+def recover(  # noqa: G3 (--journal is a read-only path; defaults to system state dir)
```

Ruff (0.15.18)

[warning] Invalid rule code in `# noqa`: G3

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)